packages/mobile/src/screens/tip-artist-screen/TipSentScreen.tsx

```diff
@@ -84,20 +83,20 @@ export const TipSentScreen = () => {
     // After success + close, take the user to the chat they were
     // attempting to make if they were unlocking DMs by tipping.
     // The saga will create the chat once the tip is confirmed
-    if (
-      source === 'inboxUnavailableModal' &&
-      account?.user_id &&
-      recipient?.user_id
-    ) {
-      dispatch(
-        chatActions.goToChat({
-          chatId: makeChatId([account.user_id, recipient.user_id])
-        })
-      )
+    if (onSuccessActions && account?.user_id && recipient?.user_id) {
```

Comment:
This is an interesting pattern. Are there any implications of order of actions, failure of an action blocking dispatch of the rest, etc?

Reply:
Yeah if an action fails/crashes the app, the next one won't be processed. I'm not in love with this pattern, it was the best idea I had though - open to alternatives!